helmfile.d/40-realtime-dashboard.yaml

```diff
@@ -0,0 +1,38 @@
+bases:
+- ../environments.yaml
+
+---
+repositories:
+  - name: radar
+    url: https://radar-base.github.io/radar-helm-charts
+  - name: cp-radar
+    url: https://radar-base.github.io/cp-helm-charts
+
+helmDefaults:
+  atomic: false
+  timeout: 240
+  kubeContext: {{ .Values.kubeContext }}
+
+releases:
+  - name: radar-jdbc-connector-agg
+    chart: radar/radar-jdbc-connector
+    version: {{ .Values.radar_jdbc_connector_agg._chart_version }}
+    installed: {{ .Values.radar_jdbc_connector_agg._install }}
+    values:
+      - {{ .Values.radar_jdbc_connector_agg | toYaml | indent 8 | trim }}
+    set:
+      - name: kafka_num_brokers
+        value: {{ .Values.kafka_num_brokers }}
+      - name: timescaledb.password
+        value: {{ .Values.timescaledb_password }}
+      - name: timescaledb.username
+        value: postgres
+      - name: timescaledb.database_name
+        value: {{ .Values.timescaledb_db_name }}
+
```

Comment:
There is already a `radar-jdbc-connector`, a second one is not needed.

Reply:
Hi Joris, the reason behind having another JDBC connector (as @yatharthranjan mentioned to me) is that because of different cases for the keys, etc in KSQL. by default KSQL converts everything to uppercase (even if the source topic has lower case), So unless we want to quote ("") everything in all the queries (to allow for case support), we need a separate connector. It is not required, it would be convenient to have.

Reply:
In that case, please add the config needed to enable this behaviour. Please note that the jdbc connector has been updated and the values are now called differently.